theforeman.org/pipelines/lib/packaging.groovy

```diff
@@ -479,7 +479,9 @@ def rsync_yum(user, ssh_key, collection, target, version) {
 
```

Comment:
the line above this (`def target_path`) is now redundant. Same for the hosts definition. In fact, it can drop the whole loop.